include/OpenSpaceToolkit/Astrodynamics/Flight/Profile.hpp

```diff
@@ -113,13 +113,18 @@ class Profile
         /// @brief Constructs a Target object.
         ///
         /// @param aType The type of the target.
-        /// @param anAxis The axis of the target.
-        /// @param isAntiDirection Whether the target is in the anti-direction.
+        /// @param aDirection The direction (unit vector) of the target.
```

Comment:
what I find confusing here, is that this argument doesn't have anything to do with the target, its about how you align the spacecraft "body" w.r.t the target?
If you supply {1,0,0} it will align the X axis of your spacecraft with the target, but if you supply {0,1,0} it will align the Y axis with it.